### ISSUE TITLE
SCRIPT ERROR in event `onResourceStart': Error: No such export SmartSigns:GetSigns in resource SmartSigns

### ISSUE THREAD

**Issue by Drift91** [SnailyCAD/live-map#40]
Hi, it seems that the SmartSigns integration has a problem that it makes SmartSigns a mandatory dependency otherwise it throws errors every time someone views the Live Map or the resource is restarted with players on the server.

```
[ script:sna-live-map] Uncaught Error: No such export SmartSigns:GetSigns in resource SmartSigns
[ script:sna-live-map] citizen:/scripting/v8/main.js(549,15): get
[ script:sna-live-map] @sna-live-map/server/server.js(21783,53):
[ script:sna-live-map] node:events(394,28): emit
[ script:sna-live-map] @sna-live-map/server/server.js(18369,22): emitReserved
[ script:sna-live-map] @sna-live-map/server/server.js(19943,14): _doConnect
[ script:sna-live-map] @sna-live-map/server/server.js(19913,18):
[ script:sna-live-map] node:internal/process/task_queues(78,11): processTicksAndRejections
[ script:sna-live-map]
[ script:sna-live-map] Error: No such export SmartSigns:GetSigns in resource SmartSigns
```

**Comment by casperiv0** [comment]
Was the SmartSigns resource updated? Since this was the correct export previously

**Comment by casperiv0** [comment]
Ah, I see. I didn’t read the description correctly, will make sure it’s an optional dependency.

**Comment by casperiv0** [comment]
This has been fixed in `0.2.2`. Please download here: https://github.com/SnailyCAD/live-map/releases/tag/0.2.2
